components/brave_wallet/browser/eth_pending_tx_tracker.cc

```diff
@@ -135,17 +134,15 @@ bool EthPendingTxTracker::ShouldTxDropped(const EthTxMeta& meta) {
     }
   }
 
-  const std::string tx_hash = meta.tx_hash();
-  if (!base::Contains(dropped_blocks_counter_, tx_hash)) {
-    dropped_blocks_counter_[tx_hash] = 0;
-  }
-  if (dropped_blocks_counter_[tx_hash] >= 3) {
-    dropped_blocks_counter_.erase(tx_hash);
+  auto [it, inserted] = dropped_blocks_counter_.try_emplace(meta.tx_hash(), 0);
```

Comment:
inserted seems unused.

Reply:
That's okay. We don't need to use that and the compiler doesn't seem to care either that it is unused.